.studio/automate-elasticsearch

```diff
@@ -28,7 +28,6 @@ document "es_delete_all_indices" <<DOC
   Delete elasticsearch indices. (Deletes every single index)
 DOC
 function es_delete_all_indices() {
-  check_if_deployinate_started || return 1
```

Comment:
I removed these because we don't need to ensure that all the Automate services are running to check Elasticsearch. I am been removing these locally for a while now.